lighthouse-core/lib/page-functions.js

```diff
@@ -113,18 +113,20 @@ function getElementsInDocument(selector) {
  */
 /* istanbul ignore next */
 function getOuterHTMLSnippet(element, ignoreAttrs = []) {
-  const clone = element.cloneNode();
-
-  ignoreAttrs.forEach(attribute =>{
-    clone.removeAttribute(attribute);
-  });
-
-  const reOpeningTag = /^[\s\S]*?>/;
-  const match = clone.outerHTML.match(reOpeningTag);
-
-  return (match && match[0]) || '';
+  try {
+    const clone = element.cloneNode();
+    ignoreAttrs.forEach(attribute =>{
+      clone.removeAttribute(attribute);
+    });
+    const reOpeningTag = /^[\s\S]*?>/;
+    const match = clone.outerHTML.match(reOpeningTag);
+    return (match && match[0]) || '';
+  } catch (error) {
```

Comment:
```suggestion
  } catch (_) {
```

we name our unused variables `_` :)